OpenRA.Mods.OpenSA/UtilityCommands/MapImporter.cs

```diff
@@ -142,5 +149,216 @@ TerrainTile GetTile(ushort tileIndex, byte frame)
 
 			return new TerrainTile(template.Id, frame);
 		}
+
+		void LoadActors()
+		{
+			for (var a = 0; a < numberOfActors; a++)
+			{
+				var id = stream.ReadInt32();
+				var type = stream.ReadInt32();
+				var x = stream.ReadInt32() / map.Grid.TileSize.Width;
+				var y = stream.ReadInt32() / map.Grid.TileSize.Height;
```

Comment:
This wont work, as it only rounds down. Original games made the move to the nearest cell. So basicaly casting the int to float, then deviding, then using Math.Round() should do the trick.